### DISCUSSION TITLE
CKEditor 5 - Upgrade feature

### DISCUSSION THREAD

**Discussion by salem84** [serenity-is/Serenity#6479] [Serenity Ideas]
Hi, I'm creating a custom editor to use CKEditor 5 in a side project of mine.
Let me know if this feature could be useful for the project, so I could create a fork of main project and update scripts (in Serenity.Assets) and HtmlContentEditor.

**Comment by VictorTomaili** [comment]
Hi @salem84
Pull requests are welcome, we will review your code and merge after everything is ok.

**Comment by volkanceylan** [comment]
CKEditor 5 license is not compatible with ours.
